.mailmap

```diff
@@ -36,6 +37,7 @@ Haiyan Meng <haiyanmeng@google.com>
 Harry Zhang <harryz@hyper.sh> <harryzhang@zju.edu.cn>
 Hu Shuai <hus.fnst@cn.fujitsu.com>
 Hu Shuai <hus.fnst@cn.fujitsu.com> <hushuaiia@qq.com>
+Iceber Gu <wei.cai-nat@daocloud.io>
```

Comment:
@Iceber Would you use your real name here? :)

Reply:
I would like to use this name.

Reply:
https://github.com/containerd/project/blob/master/CONTRIBUTING.md#sign-your-work

This one was handed down to us from the lawyers, we can't accept anonymous or pseudonym contributions. We make a best effort to enforce this to keep us away from the much more burdensome CLA model.